src/collection/collection.go

```diff
@@ -159,15 +160,19 @@ func buildSchemaListForDatabase(con *connection.PGSQLConnection) (SchemaList, er
 
 	for index, row := range dataModel {
 		if !row.SchemaName.Valid || !row.TableName.Valid {
-			log.Error("Query responded with a null schema name or table name. Skipping row %d", index)
+			if row.IndexName.Valid {
+				log.Debug("Skipping Index %s. Schema name or Table name null", row.IndexName.String)
```

Comment:
would make sense to print something like:
```
("validSchema: %b, ValidTableName: %b,",  row.SchemaName.Valid, [...])
```

To show what it is complaining about?

Reply:
Done 👍